Fabric.Authorization.API/Bootstrapper.cs

```diff
@@ -144,7 +145,7 @@ private void ConfigureSingletonRegistrations(TinyIoCContainer container)
             ConfigureSingletonDataStores(container, _appConfig);
         }
 
-        private void ConfigureSingletonDataStores(TinyIoCContainer container, IAppConfiguration appConfig)
+        private static void ConfigureSingletonDataStores(TinyIoCContainer container, IAppConfiguration appConfig)

```

Comment:
what was the reason for making this static?

Reply:
@kpaul I was just seeing a Resharper warning / suggestion to make it static.

Reply:
ahh ok. maybe it should be extracted out into an extension method